nifi-extension-bundles/nifi-py4j-bundle/nifi-py4j-bridge/src/main/java/org/apache/nifi/python/processor/RecordTransformProxy.java

```diff
@@ -264,13 +265,16 @@ private void writeResult(final RecordTransformResult result, final Map<RecordGro
             final FlowFile destinationFlowFile = session.create(originalFlowFile);
 
             final RecordSetWriter writer;
+            OutputStream out = null;
```

Comment:
Is there a reason for declaring this outside of the try-block instead of using try-with-resources? That should ensure the OutputStream is closed without the need for closeQuietly.

Reply:
Thanks for the explanation, that makes sense.